docs/docs/behaviors/sensor-rotate.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Sensor Rotation
+sidebar_label: Sensor Rotation
+---
+
+## Summary
+
+The Sensor Rotation behavior sends a different keypress, depending on whether the sensor is rotated clockwise or counter-clockwise.
```

Comment:
Maybe change "sends a different keypress" into "triggers a configured behavior"

Reply:
I think this wording makes more sense and I've added it to the pr, thanks for the suggestion!